src/Sylius/Bundle/PaymentBundle/DependencyInjection/Configuration.php

```diff
@@ -57,8 +57,23 @@ public function getConfigTreeBuilder()
             ->children()
                 ->scalarNode('driver')->defaultValue(SyliusResourceBundle::DRIVER_DOCTRINE_ORM)->end()
                 ->arrayNode('gateways')
+                    ->beforeNormalization()
+                        ->always()
+                        ->then(function ($v) {
+                            foreach($v  as $gatewayKey=>&$config)
+                            {
+                                if(!isset($config['gateway_key']))
+                                    $config['gateway_key']='sylius.payum_gateway.'.$gatewayKey;
```

Comment:
@pamil,  I create the gateway_key  because I don't mean break the old gateway translation.